_data/fellows.yml

```diff
@@ -1,15 +1,31 @@
 - name: Will
   img: logo.jpg
   location: London, UK
+  linkedin: https://www.linkedin.com/in/prakhar-rathi/
```

Comment:
Maybe instead of Will's name. We should have all the names as Prakhar only. And in future, we can add the fellows name as well when we have complete data.

Reply:
That isn't required, actually. This is for testing purposes. Issue #13 is already working on that. Have discussed with @samarthya-jha and his team will add the data to the file.

Reply:
@prakharrathi25 is correct. The scope of this issue is adding social media icons. This reduces any confusion or possibilities of merge conflicts.